tests/test_qnode.py

```diff
@@ -460,6 +461,20 @@ def circuit(w, x=None):
         self.assertAllAlmostEqual(c, [-1., -1.], delta=self.tol)
 
 
+    def test_keywordargs_for_wires(self):
+        "Tests that wires use keyword arguments."
+        self.logTestName()
+
+        def circuit(x, q=None):
+            qml.RX(x, [q])
+            return qml.expval.PauliZ(q)
+
+        circuit = qml.QNode(circuit, self.dev1)
+
+        c = circuit(np.pi, q=0)
+        self.assertAlmostEqual(c, -1., delta=self.tol)
```

Comment:
can we also add a test where the default of `q` is used? (would have to be a valid value, not `None`)

Reply:
Sure, I'll do that now